services/121-service/seed-data/instance/instance-pilot-nl.json

```diff
@@ -9,14 +9,14 @@
+    "ar": "آخر تعديل في 16\\11\\2021 نحن، <strong> الصليب الآحمر الهولندي </strong> نحتاج أن نطلب منك بعض المعلومات في الأسفل لكي نقوم بمساعدتك مادياً. سوف يتم ذلك عبر قسائم السوبر ماركت. <br><br> سوف نطلب منك <ul> <li> الإسم الأول، اسم العائلة </li> <li> v-number (إذا لم يكن لديك واحداً، الرجاء إدخال <code> 0000000000</code>) </li><li> رقم هاتف للرسائل النصية </li><li> رقم هاتف للواتس آب </li></ul> <br><br> سيتم تخزين هذه المعلومات على أجهزة الصليب الأحمر و سيتم مشاركتها فقط مع مكتب امستردام (إذا كنت مسجلاً في قائمة الإنتظار في امستردام) أو مع مكتب أوترخت (إذا كنت مسجلاً هناك) أو مع STIL (إذا كنت مسجلاً هناك) أو مع ROS (إذا كنت مسجلاً هناك) لتقديم خدمة عادلة للجميع. <br> <br> <em>Twilio</em> ستحصل على رقم هاتفك لإرسال: <ul> <li> رسائل نصية لإعلامك عن برنامج المعونات </li> <li> رسائل واتس آب لإرسال قسائم السوبر ماركت (إذا تم قبول طلبك) </li> </ul> <br><br> يمكنك الخروج من برنامج المعونات و حذف معلوماتك الشخصية في أي وقت <br> سوف نقوم بحذف معلوماتك عند نهاية المشروع. <br><br> إذا كان لديك أي اسئلة، الرجاء التواصل عبر واتس آب على <a target='_blank' rel='noopener' href='https://wa.me/31614458781'>31614458781+</a>",
+    "en": "Last updated on: 16-11-2021   <br><br>   We, <strong>The Netherlands Red Cross (Rode Kruis)</strong> need to ask you the information below in order to provide cash aid. This will be done using supermarket vouchers.   <br><br>   We will ask you your: <ul> <li>First name, last name</li><li>V-number (if you don’t have a V-number, please fill <code>0000000000</code>)</li> <li>Phone number for SMS</li> <li>Phone number for WhatsApp</li> </ul>   <br><br>   This information will be stored securely on a Rode Kruis computer and shared only with the Loket Amsterdam (if you are on the waiting list in Amsterdam) or the Aanmeldloket Utrecht and SNDVU (if you are on the waiting list in Utrecht) or STIL (if you are registered with them) or ROS (if you are registered with them) to ensure a fair process.   <br><br>   <em>Twilio</em> will receive your phone number to send: <ul> <li>SMS's to inform you on the aid program</li> <li>WhatsApp messages to send supermarket vouchers (if your application is successful)</li> </ul>   <br><br>   You can leave the aid program and delete your personal information at any time. <br> We will delete your personal information at the end of the project.   <br><br>   If you have any questions, please contact us via WhatsApp on: <a target='_blank' rel='noopener' href='https://wa.me/31614458781'>+31 6 14 45 87 81</a>",
```

Comment:
```suggestion
    "en": "Last updated on: 11-01-2022   <br><br>   We, <strong>The Netherlands Red Cross (Rode Kruis)</strong> need to ask you the information below in order to provide cash aid. This will be done using supermarket vouchers.   <br><br>   We will ask you your: <ul> <li>First name, last name</li><li>V-number (if you don’t have a V-number, please fill <code>0000000000</code>)</li> <li>Phone number for SMS</li> <li>Phone number for WhatsApp</li> </ul>   <br><br>   This information will be stored securely on a Rode Kruis computer and shared only with the Loket Amsterdam (if you are on the waiting list in Amsterdam) or the Aanmeldloket Utrecht and SNDVU (if you are on the waiting list in Utrecht) or STIL (if you are registered with them) or ROS (if you are registered with them) to ensure a fair process.   <br><br>   <em>Twilio</em> will receive your phone number to send: <ul> <li>SMS's to inform you on the aid program</li> <li>WhatsApp messages to send supermarket vouchers (if your application is successful)</li> </ul>   <br><br>   You can leave the aid program and delete your personal information at any time. <br> We will delete your personal information at the end of the project.   <br><br>   If you have any questions, please contact us via WhatsApp on: <a target='_blank' rel='noopener' href='https://wa.me/31614458781'>+31 6 14 45 87 81</a>",
```